src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -67,6 +67,22 @@ const inlineEditModeSaveSnippet = `<EuiInlineEditText
 import InlineEditValidation from './inline_edit_validation';
 const inlineEditValidationSource = require('!!raw-loader!././inline_edit_validation');
 
+import InlineEditReadOnly from './inline_edit_read_only';
+const InlineEditReadOnlySource = require('!!raw-loader!././inline_edit_read_only');
+const inlineEditReadOnlySnippet = `<EuiInlineEditText
+  inputAriaLabel="Edit text inline"
+  defaultValue="This is read only text!"
+  isReadOnly={isReadOnly}
+/>
+
+<EuiInlineEditTitle
+  inputAriaLabel="Edit title inline"
+  defaultValue="This is a read only title!"
+  heading="h3"
+  isReadOnly={isReadOnly}
+/>
+`;
```

Comment:
Quick optional note - for multiple snippets you can actually pass an array, which makes each individual section more easily copy-able
```suggestion
const inlineEditReadOnlySnippet = [
  `<EuiInlineEditText
  inputAriaLabel="Edit text inline"
  defaultValue="This is read only text!"
  isReadOnly={isReadOnly}
/>`,
  `<EuiInlineEditTitle
  inputAriaLabel="Edit title inline"
  defaultValue="This is a read only title!"
  heading="h3"
  isReadOnly={isReadOnly}
/>`,
];
```

Alternatively, we could also opt to only include a snippet for `EuiInlineEditText` and not title as well (since the `onSave` snippet already does that).
```suggestion
const inlineEditReadOnlySnippet = `<EuiInlineEditText
  inputAriaLabel="Edit text inline"
  defaultValue="This is read only text!"
  isReadOnly={isReadOnly}
/>`;
```

Reply:
I didn't know that. I'll opt for keeping only the `EuiInlineEditText` example to prevent redundant snippets.

Reply:
Perfection!